ljclang_Index_h.lua

```diff
@@ -1000,10 +1055,13 @@ enum CXTemplateArgumentKind {
  unsigned clang_isFunctionTypeVariadic(CXType T);
  CXType clang_getCursorResultType(CXCursor C);
  unsigned clang_isPODType(CXType T);
+ unsigned clang_isFinalType(CXType T);
```

Comment:
Update: generating from Clang 4.0 from the same source doesn't show these functions either. Mysterious...